Userland/Libraries/LibGfx/ImageFormats/GIFLoader.cpp

```diff
@@ -269,7 +269,7 @@ static ErrorOr<void> load_gif_frame_descriptors(GIFLoadingContext& context)
     for (;;) {
         u8 sentinel = TRY(context.stream.read_value<u8>());
 
-        if (sentinel == '!') {
+        if (sentinel == 0x21) {
```

Comment:
nit: i find the lhs easier to read. it's a meaningful ascii value, and the character is more easily to scan for in hexdump output.